docs/modules/ROOT/pages/vshn-managed/keycloak/customization.adoc

```diff
@@ -32,3 +32,133 @@ spec:
 <1> Container image url.
 <2> Secret name containing credentials for the registry.
 <3> Secret namespace
+
+
+== Configuration
+
+You can provide a custom configuration to keycloak using a `ConfigMap``.
+The content of the `ConfigMap` is a JSON file based on the keycloak export files.
```

Comment:
```suggestion
The content of the `ConfigMap` is a JSON file based on the Keycloak export files.
```

Do we have a link to somewhere which describes what a "Keycloak export file" is?

Reply:
Inventage will publish some docs on their end which documents also the default configuration they already apply and can be overwritten. This documentation will be amended once I've got their documentation.

Reply:
Perfect, that makes sense. Thanks.